resources/titlepage.cls

```diff
@@ -3,26 +3,30 @@
 \begin{center}
 
 %{\sf{\huge \sphinxDocumentTitle}} \\[5mm]
-\begin{tabularx}{\textwidth}{ |L|L|L|L|L| }
+\begin{tabularx}{\textwidth}{ |L|L|L|L|L|L| }
         \hline
-        Wydział \bf{\sphinxDocumentFaculty} &
-        \sphinxDocumentAuthor &
+	\sphinxDocumentFaculty &
+	\multicolumn{2}{|c|}{\sphinxDocumentAuthor}
+	\vline &
```

Comment:
I think I'll not change this. This word "wydział" should be left here rather in python config.